src/org/traccar/BasePipelineFactory.java

```diff
@@ -238,7 +238,9 @@ public ChannelPipeline getPipeline() {
         }
 
         if (Context.getConfig().getBoolean("forward.enable")) {
-            pipeline.addLast("webHandler", new WebDataHandler(Context.getConfig().getString("forward.url")));
+            final Boolean json = Context.getConfig().getBoolean("forward.json");
```

Comment:
I think we can safely inline the variables.

Reply:
Its longer than 150 lines, Travis CI didn't accept it

Reply:
You can separate it into several lines.

Reply:
That's what I did